nwamugo/politico
# #163755719 Ft power html api 163755719

#### What does this PR do?
Shows improvement on my test coverage results

#### Description of Task to be completed?
Get test coverage to be above 60%

#### How should this be manually tested?
Visit my github branch to see its badge
https://github.com/nwamugo/politico/tree/ft-power-html-api-163755719

#### Any background context you want to provide?
My test coverage used to be below 50%. Now it's up to 65%

#### Screenshots (if appropriate)
![capture](https://user-images.githubusercontent.com/41264503/52473070-3329a700-2b95-11e9-9de8-599a631efcf6.PNG)
